src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String lineData;
```

Comment:
Divide program logic

see https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file

Reply:
the program was divided into 3 parts
1) processing of the file. It calls two helper methods: processInputData and writeToFile
2) processing input data
3) writing to the file